nixos/modules/services/web-servers/unit/default.nix

```diff
@@ -94,22 +94,39 @@ in {
       path = with pkgs; [ curl ];
       preStart = ''
         test -f '/run/unit/control.unit.sock' || rm -f '/run/unit/control.unit.sock'
```

Comment:
We can stop removing stuff in `/run/unit/*`, now that `RuntimeDirectoryPreserve` isn't set anymore, the directory is empty during start.

Reply:
Removed